src/utils/themes-manager.js

```diff
@@ -74,10 +76,14 @@ export class ThemesManager {
    */
   setAccentOrAcknowledgementColor(colorTitle: string, color: string): void {
     const [hue, saturation, lightness] = hexToHsl(color);
-    document.querySelector(`.${style.player}`)?.style.setProperty(ACTUAL_USED_CSS_VAR.replace('{name}', colorTitle), color);
-    document.querySelector(`.${style.player}`)?.style.setProperty(HSL_HUE_CSS_VAR.replace('{name}', colorTitle), `${Math.round(hue)}deg`);
-    document.querySelector(`.${style.player}`)?.style.setProperty(HSL_SATURATION_CSS_VAR.replace('{name}', colorTitle), `${Math.round(saturation)}%`);
-    document.querySelector(`.${style.player}`)?.style.setProperty(HSL_LIGHTNESS_CSS_VAR.replace('{name}', colorTitle), `${Math.round(lightness)}%`);
+    this.playerElement.querySelector(`.${style.player}`)?.style.setProperty(ACTUAL_USED_CSS_VAR.replace('{name}', colorTitle), color);
```

Comment:
could be a case where the playerElement is null or undefined? for example, if it can't get the element by the provided targetId. if that's a case we need to handle, then you'd need to add `?` in all places where you try to access `playerElement`.

Reply:
done

Reply:
target id comes from the config, so if the target is wrong the player was not up and running in the first place 

Reply:
@JonathanTGold  right, thanks for pointing that out.
@Tzipi-kaltura so you can remove the `?`